pytest/tests/sanity/upgradable.py

```diff
@@ -164,8 +164,14 @@ def test_upgrade() -> None:
     assert 'error' not in res, res
     assert 'Failure' not in res['result']['status'], res
 
-    hex_account_balance = int(
-        nodes[0].get_account(hex_account_id)['result']['amount'])
+    # We have seen test failures, where account was not found.
+
+    response = nodes[0].get_account(hex_account_id)
+
+    if 'result' not in response or 'amount' not in response['result']:
+        logger.error("got invalid response from node %s" % response)
```

Comment:
Just `assert 'error' not in response, f'got error from node: {response["error"]}'`

Reply:
Thanks, that's much better.